frontends/pytorch/test/lit.cfg.py

```diff
@@ -58,10 +58,11 @@
 config.npcomp_tools_dir = os.path.join(config.npcomp_obj_root, 'bin')
 
 # Tweak the PATH to include the tools dir.
+npcomp_python_dir = "python" if config.npcomp_built_standalone else "tools/npcomp/python"
```

Comment:
Side comment (nothing to do about it here): I find this layering of such projects under 'tools' to be really annoying but haven't taken the time to see what would need to be done to pop it up a level.